### DISCUSSION TITLE
Share an experience in dealing with 404

### DISCUSSION THREAD

**Discussion by uiuing** [auroral-ui/hexo-theme-aurora#70] [Q&A]
After the blog was deployed in nginx, there was a 404 problem on the secondary page.

For example, /about page, refresh will show 404,Or just copy the link and open it, and 404 will also appear.

**I solved it like this**
Modify the Nginx file:
```
location /{
     error_page 404 /index.html;
}
```
Restart nginx:
`sudo service nginx restart`

**Comment by bennyxguo** [comment]
Yes if you are using nginx you need to apply a SPA rule to make sure all your sub url are appointed to the index.html, I will add this part in the document for others to take notice of this. Thanks!~

**Comment by iamthecloverly** [comment]
> After the blog was deployed in nginx, there was a 404 problem on the secondary page.
> 
> For example, /about page, refresh will show 404,Or just copy the link and open it, and 404 will also appear.
> 
> **I solved it like this** Modify the Nginx file:
> 
> ```
> location /{
>      error_page 404 /index.html;
> }
> ```
> 
> Restart nginx: `sudo service nginx restart`

I'm also facing the same issue, but deployed in vercel.

**Reply by iamthecloverly** [reply]
> > After the blog was deployed in nginx, there was a 404 problem on the secondary page.
> > For example, /about page, refresh will show 404,Or just copy the link and open it, and 404 will also appear.
> > **I solved it like this** Modify the Nginx file:
> > ```
> > location /{
> >      error_page 404 /index.html;
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Restart nginx: `sudo service nginx restart`
> 
> I'm also facing the same issue, but deployed in vercel.

@bennyxguo Is there any fix?
I can't find in docs.
